apps/web/src/components/settings/ShieldedAccount/index.tsx

```diff
@@ -89,6 +99,12 @@ export default function ShieldedAccount({ sx }: { sx: SxProps }) {
     }
   }, [alias])
 
+  const canContribute = useMemo(() => {
```

Comment:
This variable was just moved to the top (nothing was changed when it comes to the logic).